tests/testutil.c

```diff
@@ -408,7 +408,7 @@ test_util_get_system_module_name (void)
     char * result;
 
     _test_util_system_module_name = malloc (64);
-    p = popen ("find /lib -name \"libc-*.so\" | head -1 | xargs basename"
+    p = popen ("find /lib -name \"libc[-.]*so*\" | head -1 | xargs basename"
```

Comment:
Did you check whether this still picks the actual libc and not the symlink on a typical LSB system like Ubuntu?